eul/Utilities/GPU.swift

```diff
@@ -63,12 +63,18 @@ extension GPU {
                 return nil
             }
 
+            var temperature: Double? = statistics["Temperature(C)"] as? Double ?? nil
+
+            if temperature == nil || temperature == 0 {
+                temperature = SmcControl.shared.gpuProximityTemperature
+            }
+
             Print("📊 statistics", statistics)
 
             return Statistic(
                 pciMatch: pciMatch,
                 usagePercentage: usagePercentage,
-                temperature: statistics["Temperature(C)"] as? Double ?? SmcControl.shared.gpuProximityTemperature,
+                temperature: temperature,
```

Comment:
can we use `?:` here to avoid using `var`? i would prefer `let`.

Reply:
i am swift beginner，you should patch it self，thanks